eng/StyleCop.Analyzers.ruleset

```diff
@@ -0,0 +1,182 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="SyleCop Rules" ToolsVersion="15.0">
```

Comment:
Where did we get this list from?

Reply:
I've pointed @pgrawehr to our corefx StyleCop ruleset so assumed that's where it's from

Reply:
I actually started with the rule set I'm used to and then compared to the corefx set. I added special comments where the corefx has a rule enabled but here it's not (yet).

Reply:
fine as a first step, we should eventually enable at minimum the same as corefx and more as a stretch. Let's focus on enforcing this first and once we got some rules running we can iterate from there

Reply:
At the end, I'll again go trough everything and document which rules are enabled there and not here. Those won't be many.